client/src/pages/Policy.js

```diff
@@ -4,26 +4,25 @@ import Layout from "./../components/Layout";
 const Policy = () => {
   return (
     <Layout title={"Privacy Policy"}>
-      <div className="row contactus ">
-        <div className="col-md-6 ">
-          <img
-            src="/images/contactus.jpeg"
-            alt="contactus"
-            style={{ width: "100%" }}
-          />
+      <div className='row contactus'>
+        <div className='col-md-6 '>
+          <img src='/images/contactus.jpeg' alt='Privacy Policy' style={{ width: "100%" }} />
```

Comment:
Probably not an issue, but kinda just noticed it's using the same image as the contact us page